dao/src/main/java/uk/gov/moj/sdt/dao/GenericDao.java

```diff
@@ -282,16 +282,23 @@ private <D extends IDomainObject> CriteriaQuery<D> createSelectQuery(Class<D> do
 
     protected Predicate createDatePredicate(CriteriaBuilder criteriaBuilder, Root<T> root, int dataRetention) {
         Path<LocalDateTime> createdDatePath = root.get("createdDate");
+        LOGGER.debug("createDatePredicate:{}", criteriaBuilder.and(
+                criteriaBuilder.greaterThanOrEqualTo((createdDatePath), atStartOfDay(dataRetention)),
+                criteriaBuilder.lessThan((createdDatePath), atBeginningOfNextDay())));
         return criteriaBuilder.and(
-            criteriaBuilder.greaterThanOrEqualTo((createdDatePath), atStartOfDay(dataRetention)),
-            criteriaBuilder.lessThan((createdDatePath), atBeginningOfNextDay()));
+                criteriaBuilder.greaterThanOrEqualTo((createdDatePath), atStartOfDay(dataRetention)),
+                criteriaBuilder.lessThan((createdDatePath), atBeginningOfNextDay()));
     }
 
     private LocalDateTime atBeginningOfNextDay() {
-        return LocalDate.now().plusDays(1).atStartOfDay();
+        LocalDateTime beginningOfNextDay = LocalDate.now().plusDays(1).atStartOfDay();
+        LOGGER.debug("beginningOfNextDay: {}", beginningOfNextDay);
+        return beginningOfNextDay;
```

Comment:
Was this changed for debugging purposes during development? If so it may be able to be reverted to it's original form.  If it is reverted then do the same to the atStartOfDay() method.